app/commands/exercise/export_solutions_to_zip_file.rb

```diff
@@ -0,0 +1,28 @@
+require 'zip'
+
+class Exercise::ExportSolutionsToZipFile
+  include Mandate
+
+  initialize_with :exercise
+
+  def call
+    file_stream = Zip::OutputStream.write_buffer do |zip|
+      solutions.each.with_index do |solution, idx|
+        solution.latest_iteration.files.each do |iteration_file|
```

Comment:
`solution.latest_iteration `

This should be the first iteration, not the latest one (we want to see what they look like pre-mentoring)